troop/views.py

```diff
@@ -280,3 +283,65 @@ def participant_row(fields, attendance_days, participant):
                 yield 1 if field in days else ""
             else:
                 yield getattr(participant, field)
+
+
+class SendEmailView(OnlyTroopManagerMixin, generic.FormView):
+    template_name = "troop/email_form.html"
+    form_class = SendEmailForm
+
+    def get_form_kwargs(self, **kwargs):
+        form_kwargs = super().get_form_kwargs(**kwargs)
+        form_kwargs["user"] = self.request.user
+        form_kwargs["troop"] = self.request.troop
+        return form_kwargs
+
+    def get_success_url(self):
+        kwargs = {"troop_number": self.request.troop.number}
+        return reverse("troop:participant.index", kwargs=kwargs)
```

Comment:
Same a before: I think we should stay on the same page
```suggestion
        return reverse("troop:mail", kwargs=kwargs)
```